src/routes/+page.svelte

```diff
@@ -94,7 +83,7 @@
 	<div class="mt-4 ml-8 text-lg">
 		<div class="mt-4">
 			<label for="searchFor">
-				Search for the definition of:
+				Search for the definite of:
```

Comment:
Please change the `definite` word for `definition` it will be more descriptive of what is asked for the user

Reply:
This spelling error has been resolved.